calculator.feature

```diff
@@ -0,0 +1,20 @@
+Feature: windows calculator
+
+  Scenario Outline: calc automation using loop
+  
+* configure robot = { highlight: true }
+* robot { fork: 'C:/Windows/System32/calc.exe' }
+* click('Clear')
+* click(value1)&&click(operation)&&click(value2)&&click('Equals')
+#* input(value1)&&input(Key.(operation))&&input(value2)&&click('Equals')
+* match locate('#CalculatorResults').name == result
+* screenshot()
+* click('Close Calculator')
```

Comment:
we don't need to open and close the app during each iteration. So please make change accordingly